src/java/org/apache/cassandra/index/sai/disk/v1/MemtableIndexWriter.java

```diff
@@ -162,7 +162,8 @@ private long flush(DecoratedKey minKey, DecoratedKey maxKey, AbstractType<?> ter
                                                                     Integer.MAX_VALUE,
                                                                     indexContext.getIndexWriterConfig()))
             {
-                indexMetas = writer.writeAll(ImmutableOneDimPointValues.fromTermEnum(terms, termComparator));
+                ByteComparable.Version version = indexDescriptor.byteComparableVersionFor(IndexComponent.KD_TREE);
```

Comment:
kd tree?

Reply:
yes, it is `NumericIndexWriter`, which writes KD-trees.